src/style.css

```diff
@@ -0,0 +1,141 @@
+#container{
+ left: 30%;
+ position: relative;
+ top: 50px; 
+width: 30%;
+border: 1px solid wheat;
+padding: 20px 20px 0px 20px;
+/* height: 700px; */
+
+}
+
+
+.title{
+    display: block;
+    font-size: 24px;
+    
+}
+
+.refresh{
+    float: right;
+    position: relative;
+    top: 3px;
+}
+
+
+
+.inputText{
+ margin-top: 15px;
+ width: 108%;
+ height: 40px;
+ position: relative;
+ left: -20px;
+ font-size: 18px;
+ border: 1px solid wheat;
+ border-left: none;
+ border-right: none;
+ /* border-style: hidden; */
+}
+
+
+::placeholder{
+    font-style: italic;
+    position: relative;
+    left: 15px;
+}
+
+
+.todo-item{
+    text-decoration: none;
+    /* background: blue; */
+    min-height: 27px;
+    width: 105%;
+    border-bottom:  1px solid wheat;
+ list-style: none;
+ font-size: 20px;
+ padding: 13px 10px;
+ left: -20px;
+ position: relative;
+}
+
+
+.delete{
+    
+    display: none;
+    float: right;
+}
+.delete:hover{
+    
+    
+}
+.edit{
+ display: none;
+ position: relative;
+ /* top: -20px; */
+ left: -30px;
+ float: right;
+}
+
+.items{
+    display: inline-block;
+    margin-left: 15px;
+    top: 5px;
+ width: 340px;
+ /* background-color: blue; */
+    
+}
+
+.select{
+ display: block;
+ float: right;
+}
+.save{
+    float: right;  
+    display: none;
+}
+
+.delete-all{
```

Comment:
Although your code is well-redacted and functional. There are some indentation and spacing errors that should be fixable by running linters. Kindly try to apply all the linters filters to your code.